pandas/core/indexes/base.py

```diff
@@ -545,11 +545,17 @@ def is_(self, other) -> bool:
         Parameters
         ----------
         other : object
-            other object to compare against.
+            Other object to compare against.
 
         Returns
         -------
-        True if both have same underlying data, False otherwise : bool
+        bool
+            Boolean indicating if both object have same underlying data,
+            False otherwise.
+
+        See Also
+        --------
+        Index.identical : Works like Index.is_ but also check metadata.
```

Comment:
```suggestion
        identical : Works like ``Index.is_`` but also check metadata.
```

Might fix CI

Reply:
I don't really understand the build error.

I think we need to keep the `Index` in `Index.identical`.

You can also try:
```suggestion
        Index.identical : Same but also checks metadata.
```